toga/interface/dialogs.py

```diff
@@ -0,0 +1,22 @@
+def info_dialog(window, title, message):
+    raise NotImplementedError()
+
+
+def question_dialog(window, title, message):
+    raise NotImplementedError()
+
+
+def confirm_dialog(window, title, message):
+    raise NotImplementedError()
+
+
+def error_dialog(window, title, message):
+    raise NotImplementedError()
+
+
+def stack_trace_dialog(window, title, message, content, retry=False):
+    raise NotImplementedError()
+
+
+def save_file_dialog(window, title, suggested_filename, file_types):
```

Comment:
GTK will let you do *, but Cocoa is a bit picky. See https://developer.apple.com/reference/appkit/nssavepanel?language=objc . I am honestly not sure how I'd implement it, other than checking for "*" (vs a list) as a special case and not doing any of it in that instance.

wrt suggested filename, "" should suffice, and it will give ".ics" if you give it a file_types of ["ics"]...?